### DISCUSSION TITLE
Welcome to om-sharp Discussions!

### DISCUSSION THREAD

**Discussion by j-bresson** [cac-t-u-s/om-sharp#349] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Report any kind of problem
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

As a general rule:
  * Please try to make the title of the topic as clear and specific as possible

If you want to report a bug or issue:  
 * Describe your problem clearly / provide reproductible examples when relevant
 * Specify your operating system and OM# version -- make sure to try the latest OM# version before to report bugs

Some of the bug reports or wishes and suggestions discussed here will become issues in the main OM# project. When an issue is transferred to the OM# development repository, a mention will appear in the conversation: your problem is being fixed, and you can track progress from there.

Older/closed issues are still archived here: https://github.com/cac-t-u-s/om-sharp-users/issues

**Comment by andersvi** [comment]
Hi!  I'm Anders :-) 

Nice to have a low-threshold discussion forum here.

**Comment by dfcompose** [comment]
Excited to see this posting! I was getting worried that 1.8 had been abandoned
